starwars/src/components/StarWars.css

```diff
@@ -0,0 +1,29 @@
+.container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.card-container {
+    height: 500px;
+    width: 30%;
+    margin: 20px auto;
+    border-radius: 15px;
+    font-family: sans-serif;
+    color: white;
+    background-image: url('../star-wars-space.png');
```

Comment:
I really like how you used this as the background for your cards, and i like the opacity level and being able to see the background image behind your card backgrounds.